engine/api/services/http.go

```diff
@@ -20,6 +22,67 @@ import (
 // HTTPClient will be set to a default httpclient if not set
 var HTTPClient sdk.HTTPClient
 
+// DoMultiPartRequest performs an http request on a service with multipart  tar file + json field
+func DoMultiPartRequest(ctx context.Context, srvs []sdk.Service, method, path string, buf *bytes.Buffer, in interface{}, out interface{}, mods ...sdk.RequestModifier) (int, error) {
+
+	body := &bytes.Buffer{}
+	writer := multipart.NewWriter(body)
+
+	// Create tar part
+	tarPartMH := make(textproto.MIMEHeader)
+	tarPartMH.Set("Content-Type", "application/tar")
```

Comment:
how can we be sure that buffer we are writing an application/tar ?
same question for the content-disposition value.

I think this should set by the caller of this function by passing the mods (...sdk.RequestModifier) arguments